.circleci/config.yml

```diff
@@ -579,6 +579,12 @@ workflows:
           filters:
             branches:
               only: main
+      - build_and_push_staging_mtr:
+          context: cfe-civil-staging-mtr
+          filters:
+            branches:
+              only:
+                - main
```

Comment:
Does is matter that it's formatted as:

```
            branches:
              only:
                - main
```
Instead of?:

```
            branches:
              only: main
```

Reply:
Doesn't matter - `only:` accepts a string or a list of strings https://circleci.com/docs/configuration-reference/#branches